src/assets/sass/components/_chart.scss

```diff
@@ -30,6 +30,29 @@ ccl-chart {
     padding-top: 1.5rem;
   }
 
+  .chart-legend {
```

Comment:
Addressed (and merge conflicts cleared) in https://github.com/azavea/climate-change-lab/pull/193/commits/111edca6fb32af70582b2be672297f61918dbccd